app/evs_app.cpp

```diff
@@ -97,7 +97,7 @@ int main(int argc, char** argv)
 
     // Load our configuration information
     ConfigManager config;
-    if (!config.initialize("/vendor/etc/vehicle/evs_app_xt.json")) {
+    if (!config.initialize("/system/etc/automotive/evs/config.json")) {
```

Comment:
Seems like same change is duplicated in next commit "evs app: Use vendor config".